react-native/react/actions/devices.js

```diff
@@ -18,17 +21,21 @@ export function loadDevices () {
   }
 }
 
-export function generatePaperKey () {
+export function generatePaperKey () : AsyncAction {
   return function (dispatch) {
     dispatch({
       type: Constants.paperKeyLoading
     })
 
-    const incomingMap = {
+    const incomingMap : {
+      'keybase.1.loginUi.promptRevokePaperKeys': loginUi_promptRevokePaperKeys,
+      'keybase.1.secretUi.getPassphrase': secretUi_getPassphrase,
+      'keybase.1.loginUi.displayPaperKeyPhrase': loginUi_displayPaperKeyPhrase
+    } = {
       'keybase.1.loginUi.promptRevokePaperKeys': (param, response) => {
         response.result(false)
       },
-      'keybase.1.secretUi.getSecret': (param, response) => {
+      'keybase.1.secretUi.getPassphrase': (param, response) => {
```

Comment:
This changed... would LOVE if we could use the constants. maybe flow will support this later